### ISSUE TITLE
编译proto文件 出错了。。。

### ISSUE THREAD

**Issue by dgclq** [dlod-openvino/book#4]
(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>for /f %i in ('dir /b *.proto') do protoc %i --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc anchor_generator.proto --python_out=.
object_detection/protos/grid_anchor_generator.proto: File not found.
object_detection/protos/ssd_anchor_generator.proto: File not found.
object_detection/protos/multiscale_anchor_generator.proto: File not found.
anchor_generator.proto:5:1: Import "object_detection/protos/grid_anchor_generator.proto" was not found or had errors.
anchor_generator.proto:6:1: Import "object_detection/protos/ssd_anchor_generator.proto" was not found or had errors.
anchor_generator.proto:7:1: Import "object_detection/protos/multiscale_anchor_generator.proto" was not found or had errors.
anchor_generator.proto:13:5: "GridAnchorGenerator" is not defined.
anchor_generator.proto:14:5: "SsdAnchorGenerator" is not defined.
anchor_generator.proto:15:5: "MultiscaleAnchorGenerator" is not defined.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc argmax_matcher.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc bipartite_matcher.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc box_coder.proto --python_out=.
object_detection/protos/faster_rcnn_box_coder.proto: File not found.
object_detection/protos/keypoint_box_coder.proto: File not found.
object_detection/protos/mean_stddev_box_coder.proto: File not found.
object_detection/protos/square_box_coder.proto: File not found.
box_coder.proto:5:1: Import "object_detection/protos/faster_rcnn_box_coder.proto" was not found or had errors.
box_coder.proto:6:1: Import "object_detection/protos/keypoint_box_coder.proto" was not found or had errors.
box_coder.proto:7:1: Import "object_detection/protos/mean_stddev_box_coder.proto" was not found or had errors.
box_coder.proto:8:1: Import "object_detection/protos/square_box_coder.proto" was not found or had errors.
box_coder.proto:14:5: "FasterRcnnBoxCoder" is not defined.
box_coder.proto:15:5: "MeanStddevBoxCoder" is not defined.
box_coder.proto:16:5: "SquareBoxCoder" is not defined.
box_coder.proto:17:5: "KeypointBoxCoder" is not defined.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc box_predictor.proto --python_out=.
object_detection/protos/hyperparams.proto: File not found.
box_predictor.proto:5:1: Import "object_detection/protos/hyperparams.proto" was not found or had errors.
box_predictor.proto:23:12: "Hyperparams" is not defined.
box_predictor.proto:66:12: "Hyperparams" is not defined.
box_predictor.proto:126:12: "Hyperparams" is not defined.
box_predictor.proto:138:12: "Hyperparams" is not defined.
box_predictor.proto:174:12: "Hyperparams" is not defined.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc eval.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc faster_rcnn.proto --python_out=.
object_detection/protos/anchor_generator.proto: File not found.
object_detection/protos/box_predictor.proto: File not found.
object_detection/protos/hyperparams.proto: File not found.
object_detection/protos/image_resizer.proto: File not found.
object_detection/protos/losses.proto: File not found.
object_detection/protos/post_processing.proto: File not found.
faster_rcnn.proto:5:1: Import "object_detection/protos/anchor_generator.proto" was not found or had errors.
faster_rcnn.proto:6:1: Import "object_detection/protos/box_predictor.proto" was not found or had errors.
faster_rcnn.proto:7:1: Import "object_detection/protos/hyperparams.proto" was not found or had errors.
faster_rcnn.proto:8:1: Import "object_detection/protos/image_resizer.proto" was not found or had errors.
faster_rcnn.proto:9:1: Import "object_detection/protos/losses.proto" was not found or had errors.
faster_rcnn.proto:10:1: Import "object_detection/protos/post_processing.proto" was not found or had errors.
faster_rcnn.proto:29:12: "ImageResizer" is not defined.
faster_rcnn.proto:38:12: "AnchorGenerator" is not defined.
faster_rcnn.proto:45:12: "Hyperparams" is not defined.
faster_rcnn.proto:98:12: "BoxPredictor" is not defined.
faster_rcnn.proto:111:12: "PostProcessing" is not defined.
faster_rcnn.proto:126:12: "HardExampleMiner" is not defined.
faster_rcnn.proto:133:12: "ClassificationLoss" is not defined.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc faster_rcnn_box_coder.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc graph_rewriter.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc grid_anchor_generator.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc hyperparams.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc image_resizer.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc input_reader.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc keypoint_box_coder.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc losses.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc matcher.proto --python_out=.
object_detection/protos/argmax_matcher.proto: File not found.
object_detection/protos/bipartite_matcher.proto: File not found.
matcher.proto:5:1: Import "object_detection/protos/argmax_matcher.proto" was not found or had errors.
matcher.proto:6:1: Import "object_detection/protos/bipartite_matcher.proto" was not found or had errors.
matcher.proto:12:5: "ArgMaxMatcher" is not defined.
matcher.proto:13:5: "BipartiteMatcher" is not defined.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc mean_stddev_box_coder.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc model.proto --python_out=.
object_detection/protos/faster_rcnn.proto: File not found.
object_detection/protos/ssd.proto: File not found.
model.proto:5:1: Import "object_detection/protos/faster_rcnn.proto" was not found or had errors.
model.proto:6:1: Import "object_detection/protos/ssd.proto" was not found or had errors.
model.proto:11:5: "FasterRcnn" is not defined.
model.proto:12:5: "Ssd" is not defined.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc multiscale_anchor_generator.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc optimizer.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc pipeline.proto --python_out=.
object_detection/protos/eval.proto: File not found.
object_detection/protos/graph_rewriter.proto: File not found.
object_detection/protos/input_reader.proto: File not found.
object_detection/protos/model.proto: File not found.
object_detection/protos/train.proto: File not found.
pipeline.proto:5:1: Import "object_detection/protos/eval.proto" was not found or had errors.
pipeline.proto:6:1: Import "object_detection/protos/graph_rewriter.proto" was not found or had errors.
pipeline.proto:7:1: Import "object_detection/protos/input_reader.proto" was not found or had errors.
pipeline.proto:8:1: Import "object_detection/protos/model.proto" was not found or had errors.
pipeline.proto:9:1: Import "object_detection/protos/train.proto" was not found or had errors.
pipeline.proto:15:12: "DetectionModel" is not defined.
pipeline.proto:16:12: "TrainConfig" is not defined.
pipeline.proto:17:12: "InputReader" is not defined.
pipeline.proto:18:12: "EvalConfig" is not defined.
pipeline.proto:19:12: "InputReader" is not defined.
pipeline.proto:20:12: "GraphRewriter" is not defined.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc post_processing.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc preprocessor.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc region_similarity_calculator.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc square_box_coder.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc ssd.proto --python_out=.
object_detection/protos/anchor_generator.proto: File not found.
object_detection/protos/box_coder.proto: File not found.
object_detection/protos/box_predictor.proto: File not found.
object_detection/protos/hyperparams.proto: File not found.
object_detection/protos/image_resizer.proto: File not found.
object_detection/protos/matcher.proto: File not found.
object_detection/protos/losses.proto: File not found.
object_detection/protos/post_processing.proto: File not found.
object_detection/protos/region_similarity_calculator.proto: File not found.
ssd.proto:4:1: Import "object_detection/protos/anchor_generator.proto" was not found or had errors.
ssd.proto:5:1: Import "object_detection/protos/box_coder.proto" was not found or had errors.
ssd.proto:6:1: Import "object_detection/protos/box_predictor.proto" was not found or had errors.
ssd.proto:7:1: Import "object_detection/protos/hyperparams.proto" was not found or had errors.
ssd.proto:8:1: Import "object_detection/protos/image_resizer.proto" was not found or had errors.
ssd.proto:9:1: Import "object_detection/protos/matcher.proto" was not found or had errors.
ssd.proto:10:1: Import "object_detection/protos/losses.proto" was not found or had errors.
ssd.proto:11:1: Import "object_detection/protos/post_processing.proto" was not found or had errors.
ssd.proto:12:1: Import "object_detection/protos/region_similarity_calculator.proto" was not found or had errors.
ssd.proto:134:14: "Hyperparams" is not defined.
ssd.proto:22:12: "ImageResizer" is not defined.
ssd.proto:28:12: "BoxCoder" is not defined.
ssd.proto:31:12: "Matcher" is not defined.
ssd.proto:34:12: "RegionSimilarityCalculator" is not defined.
ssd.proto:45:12: "BoxPredictor" is not defined.
ssd.proto:48:12: "AnchorGenerator" is not defined.
ssd.proto:51:12: "PostProcessing" is not defined.
ssd.proto:62:12: "Loss" is not defined.
ssd.proto:160:12: "Hyperparams" is not defined.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc ssd_anchor_generator.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc string_int_label_map.proto --python_out=.

(tf_gpu) D:\vrs\tf_train\models\research\object_detection\protos>protoc train.proto --python_out=.
object_detection/protos/optimizer.proto: File not found.
object_detection/protos/preprocessor.proto: File not found.
train.proto:5:1: Import "object_detection/protos/optimizer.proto" was not found or had errors.
train.proto:6:1: Import "object_detection/protos/preprocessor.proto" was not found or had errors.
train.proto:17:12: "PreprocessingStep" is not defined.
train.proto:26:12: "Optimizer" is not defined.
